pl_bolts/models/detection/components/torchvision_backbones.py

```diff
@@ -0,0 +1,106 @@
+
+import torch.nn as nn
+from typing import Optional
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+try:
+    import torchvision
+
+    from ._supported_models import TORCHVISION_MODEL_ZOO
+except ModuleNotFoundError:
+    warn_missing_pkg('torchvision')  # pragma: no-cover
+
+
+__all__ = ["create_torchvision_backbone"]
```

Comment:
```suggestion
```
not sure if this is needed, the `__all__` serves only for `from abc.fgh import *`